library/Vanilla/Formatting/Embeds/ImgurEmbed.php

```diff
@@ -28,7 +28,7 @@ public function __construct() {
      */
     public function matchUrl(string $url) {
         $data = [];
-
+        $url = str_replace(['.jpg', '.jpeg', '.png'], '', $url);
```

Comment:
This seems to work, but could we get an inline comment on this explaining the situation. Either that or maybe list the types of URL structures supported in the docblock above?